source/auth/tests/connection-string.yml

```diff
@@ -85,6 +85,10 @@ tests:
             mechanism: "GSSAPI"
             mechanism_properties:
                 SERVICE_NAME: "mongodb"
+    -
+        description: "should raise an error when the authSource is empty"
```

Comment:
Per the spec language change, this test should be renamed to "must raise an error..." (along with the JSON version).